FortranCon2020-stdlib/README.md

```diff
@@ -0,0 +1,37 @@
+# FortranCon2020-stdlib
+
+**Authors**:
+[Balint Arady](https://github.com/aradi)
+[Izaak Beekman](https://github.com/zbeekman)
+[Neil Carlson](https://github.com/nncarlson)
+[Ondrej Certik](https://github.com/certik)
+[Pedro Costa](https://github.com/p-costa)
+[Milan Curcic](https://github.com/milancuric)
+[Pierre de Buyl](https://github.com/pdebuyl)
+[???](https://github.com/fiolj)
+[???](https://github.com/JHenneberg)
+[Michael Hirsch](https://github.com/scivision)
+[Ivan Pribec](https://github.com/ivan-pi)
+[???](https://github.com/sakamoti)
+[???](https://github.com/nshaffer)
+[Jeremie Vandenplas](https://github.com/jvdp1)
+[Ashwin Vishnu](https://github.com/ashwinvis)
+
+
+**Track**: Junior presentations (15 + 5 minutes discussion)
+
+**Presenter**: To be confirmed
+
+**Title**: Fortran Standard Library
+
+**Alternative title**: Fortran Standard Library - a community project
+
+**Abstract**:
+The Fortran Standard, as published by the International Organization for Standardization (ISO), does not include a Standard Library.
+Therefore, the goal of this project is to provide a community driven and agreed upon de facto "standard" library for Modern Fortran, called the Fortran Standard Library (stdlib; https://github.com/fortran-lang/stdlib).
+This library aims to provide to the community a set of procedures for science, engineering, and mathematics.
+The overall scope of the Fortran Standard Library is therefore similar to the one of SciPy or to the default built-in Matlab scientific environment.
+Started a few months ago, no less than 15 people already contributed to the development of the Fortran Standard Library and its documentation.
+Currently the library includes procedures for catching and handling errors, handling optional arguments, facilitating I/O operations, linear algebra, numerical integration, and descriptive statistics.
+Many other programmers are also involved in active discussions about the development of the Fortran Standard Library through GitHub issues.
+The development of this library is part of the Fortran-lang project and aims to collaborate with the Fortran Standards Committee.
```

Comment:
Do we want to be more specific? My understanding is that stdlib is to be the reference implementation for compiler vendors if is eventually adopted by the standards committee.

Reply:
I keep your suggestion for the presentation. I can't introduce this in the abstract due to word limits